src/services/dashboards/dashboard-detail/modules/DashboardWidgetContainer.vue

```diff
@@ -108,7 +108,7 @@ export default defineComponent({
             observeInstance.observe(containerRef?.value as Element);
         });
 
-        onUnmounted(() => {
```

Comment:
This is WidgetContainer. 
When I remove a dashboard. some error has occurred.

On `onMounted`, element has removed already.
So, `onBeforeUnmount` hooks is correct .